features/generate-6.feature

```diff
@@ -16,4 +16,15 @@ Scenario: The project is an iOS application with core data models (ios_app_with_
     Then tuist generates the project
     Then I should be able to build for iOS the scheme App
     Then the product 'App.app' with destination 'Debug-iphonesimulator' contains resource 'Users.momd'
-    Then the product 'App.app' with destination 'Debug-iphonesimulator' contains resource '1_2.cdm'
\ No newline at end of file
+    Then the product 'App.app' with destination 'Debug-iphonesimulator' contains resource '1_2.cdm'
+
+Scenario: The project is an iOS application with appclip (ios_app_with_appclip)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture ios_app_with_appclip into the working directory
+    Then tuist generates the project
+    Then in project App the target App should contain the build phase Embed App Clips
```

Comment:
This test doesn't add that much value because asserting that the build phase exists doesn't imply that the build phase is valid. I'd rather build the app, and then inspect whether the clip has been copied into the final product. That gives us much more confidence about the build phase being right. Wi have a similar step for testing that the dynamic frameworks have been successfully copied with the unnecessary archs stripped.